### ISSUE TITLE
[Bug]:  Inconsistent CTM

### ISSUE THREAD

**Issue by WenXin20** [XFactHD/FramedBlocks#235]
### Minecraft version

1.19

### Forge version

`v43.2.3`

### FramedBlocks version

`v6.7.0`

### Describe the bug

Framed panels don't have connected textures on the inner half of the block, so when used for windows, one side won't have ctm.

### How to reproduce the bug

1. Place Framed Panels
2. Use Glass Blocks
3. Installed [MC Connected CTM](https://www.curseforge.com/minecraft/texture-packs/mc-connected-ctm)

### Expected behavior

Both sides should have ctm enabled.

### Additional details

![image](https://user-images.githubusercontent.com/12000593/221763148-c5dc8ee3-e8af-46f0-8309-f8dde392674a.png)

![image](https://user-images.githubusercontent.com/12000593/221763343-0724d138-232d-4d89-ace6-01f9c19dfa4f.png)

**Comment by XFactHD** [comment]
6.7.0 contains a major rework of how I handle connected textures support, which includes removing support for CTM's proprietary facade system (allows them to ask my blocks what they actually look like) and replacing it with Forge's improved implementation of such a system. CTM however has yet to implement support for this new system (https://github.com/Chisel-Team/ConnectedTexturesMod/issues/173#issuecomment-1416032110). The reason I decided to remove it immediately was the major maintenance burden it had.
TL;DR: Support for CTM is broken at the moment and fails spectacularly under certain conditions:
![grafik](https://user-images.githubusercontent.com/11262040/221925231-50894b7c-e99a-4971-b8ef-7f042f3c71d1.png)

As for the actual question at hand and ignoring the broken CTM support: the improvements made in the aforementioned rework include preliminary support for connected textures on faces that aren't on the outer edge of the block and/or don't cover the full area of the face (the back sides of the panels in your screenshot are an example of the former and their sides and top and bottom are examples of the latter). The behaviour of that isn't perfect yet though and currently causes connections towards blocks it shouldn't connect to, which is one reason why the config for this currently defaults to only work on the faces where your screenshot shows it working. Unfortunately however, at this point in time I don't see any way of supporting CTM in this extended connected textures support because I have no way of accessing the data that identifies a unique combination of connections to neighbors, which I need in order to cache the quads you see in the end, so only the faces that already appear to be working now will also work when CTM implements support for Forge's appearance API.

I'll keep this issue open until CTM at least properly works on the parts that already appear to work. I won't hold my breath for the extended support though.

**Comment by WenXin20** [comment]
Ah I see, thanks for the reply

**Comment by XFactHD** [comment]
Connected textures support has been significantly improved in the 1.20.1 update which was just released. Due to the amount and type of changes, this can unfortunately not be backported to 1.19.2. Issues like the connection weirdness in my screenshot above will have to be dealt with on a case-by-case basis with the individual CT mods (CTM, Create, XyCraft, Fusion and Athena are the ones I'm aware of). As such, I'll close this issue for now and deal with the individual issues as they come up.

**Comment by WenXin20** [comment]
Sounds good. Interesting to see so many new ctm mods now. Connectedness is another
